apps/web/src/utils/client.ts

```diff
@@ -37,3 +37,32 @@ export const { publicClient, chains } = configureChains(
     pollingInterval: 6_000,
   },
 )
+
+export const { publicClient: publicDelicateClient } = configureChains(
+  CHAINS,
+  Array.from({ length: 1 })
+    .map((_, i) => i)
+    .map((i) => {
+      return jsonRpcProvider({
+        rpc: (chain) => {
+          if (process.env.NODE_ENV === 'test' && chain.id === mainnet.id && i === 0) {
+            return { http: 'https://ethereum.publicnode.com' }
+          }
+          return PUBLIC_NODES[chain.id]?.[i]
+            ? {
+                http: PUBLIC_NODES[chain.id][i],
+              }
+            : null
+        },
+      })
+    }),
+  {
+    batch: {
+      multicall: {
+        batchSize: 1024 * 200,
+        wait: 16,
+      },
+    },
+    pollingInterval: 6_000,
+  },
+)
```

Comment:
So the issue we talked about also occurs when we use the same rpc endpoint right? If so there's no need for dedicated endpoint? Shall we just remove this part?

Reply:
sure, have tested, even same rpc will have same issue, let me remove these loc